packages/gatsby-source-wordpress/README.md

```diff
@@ -600,6 +606,52 @@ Full example:
 }
 ```
 
+### Query pages with the Yoast Fields Node
+
+```graphql
+{
+  allWordpressPage {
+    edges {
+      node {
+        yoast_meta {
+          yoast_wpseo_title
+          yoast_wpseo_metadesc
+          yoast_wpseo_canonical
+          yoast_wpseo_facebook_title
+          yoast_wpseo_facebook_description
+          yoast_wpseo_facebook_type
+          yoast_wpseo_facebook_image
+          yoast_wpseo_twitter_title
+          yoast_wpseo_twitter_description
+          yoast_wpseo_twitter_image
+          yoast_wpseo_social_url
+          yoast_wpseo_company_or_person
+          yoast_wpseo_person_name
+          yoast_wpseo_company_name
+          yoast_wpseo_company_logo
+          yoast_wpseo_website_name
+        }
+      }
+    }
+  }
+}
+```
+
+### Query Yoast Redirects
+
+**_only work with Yoast Premium_**
+
+```graphql
+{
+  wordpressYoastRedirects {
```

Comment:
I would suggest using

```graphql
{
  allWordpressYoastRedirects {
    edges {
      node {
        origin
        url
        type
        format
      }
    }
  }
}
```

to get all redirects and not just one

Reply:
Voilà 😁